sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -181,3 +183,31 @@ def from_dict(cls, event):
                     " The `source` and `type` params are required."
                     )
         return event_obj
+
+    @staticmethod
+    def _get_bytes(obj):
+        """Event mixin to have methods that are common to different Event types
+        like CloudEvent, EventGridEvent etc.
+        """
+        try:
+            # storage queue
+            return json.loads(obj.content)
+        except ValueError:
+            raise ValueError(
+                "Failed to retrieve content from the object. Make sure the "
+                 + "content follows the CloudEvent schema."
```

Comment:
Technically this error will be raised if the content isn't valid JSON, regardless of whether that JSON follows the schema?

Reply:
That is correct -the wording is a bit intentional